bin/update-dependencies

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -eu
+
+npm outdated | cut -d ' ' -f1 | tail -n +2 | xargs -L1 -I {}  npm install '{}@latest'
```

Comment:
I was using `ncu -u` instead
`https://www.npmjs.com/package/npm-check-updates`
Can you keep using it or your bash script is somehow better?